src/stores/notification.ts

```diff
@@ -1,10 +1,5 @@
-import {
```

Comment:
todo: the export of this file still uses a custom type (`<typeof ...`). Please align with your changes from `breadcrumb.ts`.